server/Makefile

```diff
@@ -0,0 +1,19 @@
+REPORTER ?= spec
+NODE_BIN := $(shell npm bin)
+BASE_DIR := $(NODE_BIN)/../..
+NODE_PATH := $(BASE_DIR)/test:$(BASE_DIR)/server:$(BASE_DIR)/client
+BABEL := $(NODE_BIN)/babel
+
+test: i18n/test/out
+	@NODE_ENV=development NODE_PATH=$(NODE_PATH) $(BASE_DIR)/test/runner.js --reporter=$(REPORTER) $(addprefix --grep ,$(GREP))
+
+test-%: i18n/test/out
+	@GREP="$*" make test
+
+i18n/test/out: i18n/test/examples
```

Comment:
Shouldn't we clean dir after each test run? We had that before: https://github.com/Automattic/wp-calypso/blob/d3fa3833a7aa9f27e7c67254c8276d11c6963947/server/i18n/Makefile.